nipype/interfaces/mrtrix3/utils.py

```diff
@@ -266,7 +267,7 @@ class ComputeTDIInputSpec(CommandLineInputSpec):
         desc='specify output image data type')
     use_dec = traits.Bool(argstr='-dec', desc='perform mapping in DEC space')
     dixel = File(
-        'dixels.txt',
+        'dixels.txt', usedefault=True,
```

Comment:
It's not clear to me from the [docs](https://mrtrix.readthedocs.io/en/latest/reference/commands/tckmap.html) that this is set by default.

Reply:
we can remove `usedefault=True`, but I'd also remove the filename, imo it might me misleading